aws-lambda-java-runtime-interface-client/pom.xml

```diff
@@ -56,7 +56,7 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-lambda-java-serialization</artifactId>
-      <version>1.0.0</version>
+      <version>1.1.0</version>
```

Comment:
After releasing the new serialization-lib we should update the dependency here

Reply:
yes, will do a separate PR once the library is published to maven